engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/repository-details-info.controller.js

```diff
@@ -14,194 +14,208 @@
-                });
-                results.unshift({
-                    id: 'noarch',
-                    name: translate('Default'),
-                    value: null
+                return deferred.promise;
+            };
+
+            $scope.architectures = function () {
+                var deferred = $q.defer();
+                Architecture.queryUnpaged(function (architectures) {
+                    var results = architectures.results;
+                    results.map(function(i) {
+                        i.id = i.name;
+                    });
+                    results.unshift({
+                        id: 'noarch',
+                        name: translate('No restriction'),
+                        value: null
+                    });
+                    deferred.resolve(results);
                 });
-                deferred.resolve(results);
-            });
-            return deferred.promise;
-        };
+                return deferred.promise;
+            };
 
-        $scope.save = function (repository, saveUpstreamAuth) {
-            var deferred = $q.defer();
+            $scope.save = function (repository, saveUpstreamAuth) {
+                var deferred = $q.defer();
 
-            if (!saveUpstreamAuth) {
-                delete repository.upstream_username;
-                delete repository.upstream_password;
-            }
+                if (!saveUpstreamAuth) {
+                    delete repository.upstream_username;
+                    delete repository.upstream_password;
+                }
 
-            if (!_.isEmpty(repository.commaTagsWhitelist)) {
-                repository["docker_tags_whitelist"] = repository.commaTagsWhitelist.split(",").map(function(tag) {
-                    return tag.trim();
-                });
-            } else {
-                repository["docker_tags_whitelist"] = [];
-            }
-
-            repository.$update(function (response) {
-                deferred.resolve(response);
-                if (!_.isEmpty(response["docker_tags_whitelist"])) {
-                    repository.commaTagsWhitelist = repository["docker_tags_whitelist"].join(", ");
+                if (!_.isEmpty(repository.commaTagsWhitelist)) {
+                    repository["docker_tags_whitelist"] = repository.commaTagsWhitelist.split(",").map(function(tag) {
+                        return tag.trim();
+                    });
                 } else {
-                    repository.commaTagsWhitelist = null;
+                    repository["docker_tags_whitelist"] = [];
                 }
-                Notification.setSuccessMessage(translate('Repository Saved.'));
-            }, function (response) {
-                deferred.reject(response);
-                _.each(response.data.errors, function (errorMessage) {
-                    Notification.setErrorMessage(translate("An error occurred saving the Repository: ") + errorMessage);
+                /* eslint-disable camelcase */
```

Comment:
Does this need to get re-enabled?

Reply:
The first question that comes to my mind is 'why aren't we just disabling the line?' The answer is that whatever cursed version of eslint is being used here doesn't seem to work with `eslint-disable-next-line`.  `eslint-disable` is used already further down on line 165, so I'd argue for just keeping the first of the two occurrences (here) and calling it good?  We use snake case far too often in these controllers to worry about disabling/enabling it each time.

Reply:
Lol. Fair!